_pages/dev/ssr/server-side-rendering-contextual-logging.md

```diff
@@ -236,17 +221,19 @@ import { ErrorHandlingModule } from '@spartacus/setup/ssr';
 export class SpartacusModule {}
 ```
 
-By importing `ErrorHandlingModule`, `CxErrorHandler` will handle all errors that occur during Server Side Rendering, and these errors will be logged with an appropriate context.
+By importing `ErrorHandlingModule`, the `CxErrorHandler` then handles all errors that occur during server-side rendering, and these errors are logged with an appropriate context.
+
+**Note:** Since user applications may contain their own implementations of `ErrorHandler`, the `CxErrorHandler` is not used by default when contextual logging is enabled. To use the `CxErrorHandler`, it is recommended that you extend your error handler to use the `LoggerService`.
 
-**Caution:** Since user applications may contain their implementations of `ErrorHandler`, `CxErrorHandler` is not used by default when contextual logging is enabled. To use it, it is recommended to utilize the `LoggerService` by extending your error handler with its usage.
+## LoggerService and Third-party Party Libraries
 
-## LoggerService And The Third-party Party Libraries
+To ensure that you have context and proper formatting for the logs that are output by third-party libraries in your application, it is recommended that you verify whether custom loggers can be provided, and that you use the `LoggerService` if possible. Otherwise, the logs from third-party libraries will be written in plain text, without the request's context and without proper formatting.
 
-`LoggerService` can be used not only to log messages added to our applications but also to those third-party libraries used in the project that want to communicate with the developers by their state. 
+The `LoggerService` can be used not only to log messages that are added to your applications, but also to log messages that are added to those third-party libraries that are used in your project and that communicate by their state.
 
-For example, Composable Storefront uses `LoggerService` to display logs created by the `i18next` library. This was achieved by creating a compatible plugin with the `i18next` configuration and using the capabilities provided by our logging service.
+For example, Spartacus uses the `LoggerService` to display logs created by the `i18next` library. This was achieved by creating a compatible plugin with the `i18next` configuration and using the capabilities provided by the Spartacus logging service.
```

Comment:
`query: ` Is it ok to use `Spartacus` in the new docs? 😄

Reply:
Yes, because my conversion script takes care of converting these to a product name variable that is set to "composable storefront" in dita cms -- whereas if we leave them as simply the string "composable storefront", the script in its current form doesn't convert these to the product variable in dita cms. I should obviously update my script, but this is the first time I've encountered this scenario 😄